src/app/zap-templates/zcl/data-model/chip/mode-base-cluster.xml

```diff
@@ -0,0 +1,103 @@
+<?xml version="1.0"?>
+<!--
+Copyright (c) 2021 Project CHIP Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<configurator>
+  <domain name="CHIP"/>
+
+<!--  <enum name="StatusCode" type="ENUM8">-->
```

Comment:
why are the StatusCode, ModeTag and the cluster xml block commented?

Reply:
Probably because they are not actually used for anything?  But the comments should say that.

Reply:
I left it in there as a template for anyone wanting to create a new derived cluster. I will add a comment to note this.

Reply:
I have added a comment about this.